README.md

```diff
@@ -97,6 +66,8 @@ The following components are provided by the customers:
 ```
 configured in the main.yml
```

Comment:
Maybe highlight that it means top-level (play.yml) dir vars/main.yml instead of the per-role one?

Reply:
Explained the entire path